src/action/authActions/authActions.js

```diff
@@ -0,0 +1,37 @@
+// import makeRequest from '../Utils/axiosSetup';
+import axios from 'axios';
+
+import SubmissionErrorMessage from '../../Utils/submissionErrorResponse';
+
+export const LOGIN_USER = 'LOGIN_USER';
+
+export const loginUser = (values, history) => async dispatch => {
+  try {
+    const response = await axios.post('https://neon-ah-staging.herokuapp.com/api/v1/auth/login', {
+      user: values.user,
+      password: values.password
+    });
+    // makeRequest('/auth/login', {
+    //   method: 'POST',
+    //   data: {
+    //     user: values.user,
+    //     password: values.password
+    //   }
+    // });
+    localStorage.setItem('User', JSON.stringify(response.data));
+
+    dispatch({
+      type: LOGIN_USER,
+      payload: response.data.data
+    });
+
+    history.push('/welcome');
+  } catch (error) {
+    const errorResponse = JSON.stringify(error.response.data.data.message);
+    if (errorResponse === '"Your account has not been verified"') {
+      SubmissionErrorMessage('You are yet to verify your account, please check your email');
```

Comment:
Hi, sola, kindly give a second look to the `SubmissionErrorMessage` variable. If it is not a class it should not be `CamelCased`